lib/vagrant-openshift/constants.rb

```diff
@@ -55,6 +55,9 @@ def self.repos_for_name(reponame)
           'origin-aggregated-logging' => aggregated_logging_repos,
           'origin-metrics' => {
             'origin-metrics' => 'https://github.com/openshift/origin-metrics.git'
+          },
+          'openshift-ansible' => {
+            'openshift-ansible' => 'https://github.com/rhcarvalho/openshift-ansible.git'
```

Comment:
Yeah, this is one of the urls we'll need to change whenever we find a long term home for this code.